scripts/RegisterDevice.py.Multi

```diff
@@ -17,7 +17,7 @@ def runCmd(cmd, wait=True):
 def setupPN532():
     answer = input('Please make sure that the PN532 reader is wired up correctly '
                    'to the GPIO ports before continuing...\n Continue?: [Y/n]')
-    if not answer or answer[0] != 'Y':
+    if answer and answer[0].lower() == 'n':
```

Comment:
Not sure, if I understand the change correctly, but it looks like checking valid answer only for no kind of reverts the logic?